### ISSUE TITLE
Installation problem: 'google' module not found

### ISSUE THREAD

**Issue by ywlds** [quantumlib/Cirq#718]
when import cirq, it pull "google" module not found? can you give me some hints how to deal with this?

**Comment by dlyongemallo** [comment]
Short answer: run `pip install -r requirements.txt`

I ran into this message also when I tried to run some of the scripts intended for development, but with only the regular (non-dev) [setup instructions](https://cirq.readthedocs.io/en/latest/install.html#installing-on-linux) completed. 

Incidentally, the link to "development environment" on that page is broken, which is issue #850. I missed the dev setup instructions because of this, which is probably also what happened with @ywlds above. 

I don't remember what sequence of commands I ran to get the error, but basically I just tried running something, and whatever missing module it complained about, I installed it. Eventually, I got the "module google not found" message and was unable to proceed further. I then realised that there was actually an additional page for dev setup. Running `pip install -r requirements.txt` fixed the problem.
